tests/benchdnn/conv/conv.cpp

```diff
@@ -667,6 +668,47 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
     if (prb->alg == WINO) {
         if (is_cpu()) {
 #if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_AARCH64
+            bool pad_ok_f32 = prb->pw <= 1 && prb->ph <= 1 && prb->pw_r <= 1
+                    && prb->ph_r <= 1;
+            bool shape_ok = prb->ndims == 4 && prb->g == 1 && prb->kh == 3
+                    && prb->kw == 3 && prb->sh == 1 && prb->sw == 1
+                    && prb->dh == 0 && prb->dw == 0 && !prb->has_groups
+                    && pad_ok_f32;
+
+            const bool dir_ok = (prb->dir & FLAG_FWD);
+
+            const auto &po = prb->attr.post_ops;
+
+            // "true" here stands for eltwise.scale == 1.f check
+            const auto is_eltwise
+                    = [&](int idx) { return po.entry[idx].is_eltwise_kind(); };
+            auto is_sum = [&](int idx) { return po.entry[idx].is_sum_kind(); };
+
+            const bool sum_with_eltwise
+                    = (po.len() == 2) && is_sum(0) && is_eltwise(1);
+            const bool eltwise_only = (po.len() == 1) ? is_eltwise(0) : false;
+            bool eltwise_ok = false;
+
+            // Compute Library supports only one eltwise post-op or
+            // sum+eltwise post-ops
+            if (eltwise_only || sum_with_eltwise) {
+                const auto act_type = po.entry[sum_with_eltwise].eltwise.alg;
```

Comment:
I think you should access the `po.entry[]` with an integral variable, not with a boolean. Your code works, but if for any reason you need to go past indexes 0 and 1 in the future, this code will need to change anyway.

Reply:
I don't think there ever will be a case (for checking Compute Library post-ops) where we check more than indexes 0 and 1.